app/config/config.yml

```diff
@@ -2,9 +2,7 @@ imports:
     - { resource: parameters.yml }
     - { resource: security.yml }
     - { resource: @SumoCodersFrameworkErrorBundle/Resources/config/config.yml }
```

Comment:
Van de core-bundle lukt het niet. Ik vermoed omdat die als eerste moet ingeladen worden en dat dmv DI de volgorde niet te beïnvloeden is? Eventueel kan jij eens kijken?